From d43feb2a69d0ca8e3b3b22be2eb32263c9364b62 Mon Sep 17 00:00:00 2001
From: Zhong Hongbo <hongbo.zhong@windriver.com>
Date: Fri, 10 Oct 2014 15:19:51 +0800
Subject: [PATCH 1813/1825] time: export clk_disable and clk_enable function

ERROR: "clk_disable" [drivers/char/hw_random/atmel-rng.ko] undefined!
ERROR: "clk_enable" [drivers/char/hw_random/atmel-rng.ko] undefined!

Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 arch/arm/mach-armada38x/time.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-armada38x/time.c b/arch/arm/mach-armada38x/time.c
index 5339dd8..953e7a4 100644
--- a/arch/arm/mach-armada38x/time.c
+++ b/arch/arm/mach-armada38x/time.c
@@ -155,11 +155,13 @@ int clk_enable(struct clk *clk)
 {
 	return 0;
 }
+EXPORT_SYMBOL(clk_enable);
 
 void clk_disable(struct clk *clk)
 {
 	/* Empty */
 }
+EXPORT_SYMBOL(clk_disable);
 
 #ifdef CONFIG_HAVE_ARM_TWD
 struct clk {
-- 
1.7.5.4

